docs/docs/developers/updates.md

```diff
@@ -97,7 +97,7 @@ function updateScale(chart) {
 }
 ```
 
-Code sample for updating options can be found in [toggle-scale-type.html](./../../../samples/latest/scales/toggle-scale-type.html).
+Code sample for updating options can be found in [toggle-scale-type.html](https://www.chartjs.org/samples/latest/scales/toggle-scale-type.html).
```

Comment:
This goes to different version, it could be confusing. Can't get the relative path to work?

Reply:
Did not look into it that much, copied it from another place where it links to the samples page in this way.

I thought the version was done on purpose so that when V3 releases it doesn't have to be adjusted again.

Reply:
Right, I missed the fact that the relative path links to latest too.